models/commit_status.go

```diff
@@ -293,8 +274,8 @@ func HashCommitStatusContext(context string) string {
 // CommitStatusContext represents commit status context
 type CommitStatusContext struct {
 	ID          int64
-	RepoID      int64          `xorm:"index"`
-	ContextHash string         `xorm:"varchar(40) unique"`
+	RepoID      int64          `xorm:"index unique(s)"`
+	ContextHash string         `xorm:"varchar(40) unique(s)"`
```

Comment:
Maybe I'm getting this wrong? But a context is the string that gets reported by a CI or other status providers. Perhaps the ci reports  the same status for multiple commits in the same repo. The `unique` modifier will prevent this, wouldn't it?